src/views/statutory/VotesAmounts.vue

```diff
@@ -71,14 +74,39 @@
 <script>
 import { mapGetters } from 'vuex'
 
+/**
+ * Takes a list of antennas, and returns an object with two properties, CSV content and a filename
+ * @param antennae
+ * @param eventShortLink the string 'aaa' in the URL http://my.appserver.test/statutory/aaa/votes. Since it's part of the URL, it does comport spaces.
+ */
+function prepareExportCsv (antennae, eventShortLink) {
```

Comment:
this should go either as a method, or, even better, you can put it as a computed property

Reply:
I wanted to reduce the amount of symbols exported, and, if possible, make it testable somewhere else.
computed property? Can you please elaborate?

Reply:
```
{
    computed: {
        csvContent() {
            let contentT = 'data:text/csv,'
            contentT += 'BODY_CODE%2CVOTES%0A'

            for (const antenna of this.antennae) {
                contentT += antenna.body.code + '%2C' + antenna.votes + '%0A'
            }
            return contentT
        },
        csvFilename() {
            return 'exportVotes_' + this.$route.params.id + '.csv' // todo  add a date to the export?
        }
    }
}
```

something like this can work, then you can do

```
<div> (ofc, add an icon here, and hide this block as long as the data is not loaded) <a :href="csvContent" :download="csvFilename">download list of votes</a> </div>
```

Reply:
Okay, so I tried this but it didn't really work for some reason.